### ISSUE TITLE
Open Radiant and/or State file from Desktop

### ISSUE THREAD

**Issue by josh1400** [radiant-rstats/radiant#93]
Is there a way to create a shortcut that opens RStudio, Radiant, and a State File in one click?

**Comment by vnijs** [comment]
I agree that would be nice. I just opened an issue/feature request about this with Rstudio on Github:

https://github.com/rstudio/rstudio/issues/3812

You can have someone run a command like the below from RStudio to open a state file in Radiant

```
radiant::radiant("https://github.com/radiant-rstats/docs/raw/gh-pages/examples/demo-dvd-rnd.state.rda")
```
